### DISCUSSION TITLE
telegram-upload in Google Colab

### DISCUSSION THREAD

**Discussion by MrNaman** [Nekmo/telegram-upload#158] [Ideas]
By installing telegram-upload in Google Colab I can easily upload my Gdrive files to a telegram channel. I hope #140 gets implemented soon so that I can backup my Gdrive data on telegram easily with _speed_

**Comment by kairin** [comment]
hi @MrNaman 

how do you install into Google Colab?

must your Colab account be a paid account or a free account works too?
